src/biblenotify/core/notifications.py

```diff
@@ -96,7 +98,43 @@ def loadChapter(self, location: str) -> dict:
 
         contents_json = json.loads(contents_string)
 
+        text = self.highlightVerses(contents_json)
```

Comment:
Okay, refactor time. 🙂 
1. Assign ``contentsJson["read"][0]["text"]`` to a variable like ``chapterText``. This will allow us to pass it into ``highlightVerses`` and be DRY (Don't Repeat Yourself)
2. Here we will want to pass in the chapter text (``chapter``) and the ``verse`` we want to highlight.
3. I would rename ``text`` to ``highlightedChapter`` for clarity.

Reply:
Ok